src/main/java/frc/robot/utils/TrigonPIDController.java

```diff
@@ -0,0 +1,57 @@
+package frc.robot.utils;
+
+import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpiutil.math.MathUtil;
+
+public class TrigonPIDController extends PIDController {
+	/**
+	 * This is TRIGONS PIDController class with a friendlier user interface for the
```

Comment:
interface - don't use that word

Reply:
change TRIGONS to TRIGON's or something similar that makes sense